### PR TITLE
Cav 444

### PR DESCRIPTION
**Pull request by mar36tin**

### CODE REVIEW

**Review by brdgeth — DISMISSED** [review]

File: app/uk/gov/hmrc/cipphonenumberverification/services/PasscodeService.scala [hunk 1]
```diff
@@ -26,14 +26,14 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class PasscodeService @Inject()(passcodeCacheRepository: PasscodeCacheRepository)
                                (implicit ec: ExecutionContext) extends Logging {
-  def persistPasscode(phoneNumberAndOtp: PhoneNumberPasscodeData): Future[PhoneNumberPasscodeData] = {
-    logger.debug(s"Storing phoneNumberAndOtp in database for ${phoneNumberAndOtp.phoneNumber}")
-    passcodeCacheRepository.put(phoneNumberAndOtp.phoneNumber)(DataKey("cip-phone-number-verification"), phoneNumberAndOtp)
-      .map(_ => phoneNumberAndOtp)
+  def persistPasscode(phoneNumberAndPasscode: PhoneNumberPasscodeData): Future[PhoneNumberPasscodeData] = {
+    logger.debug(s"Storing phoneNumberAndPasscode in database for ${phoneNumberAndPasscode.phoneNumber}")
```
Comment (johno1985) [inline app/uk/gov/hmrc/cipphonenumberverification/services/PasscodeService.scala:30]: drop this as it is PII

File: app/uk/gov/hmrc/cipphonenumberverification/services/PasscodeService.scala [hunk 2]
```diff
@@ -26,14 +26,14 @@ import scala.concurrent.{ExecutionContext, Future}
...
+    passcodeCacheRepository.put(phoneNumberAndPasscode.phoneNumber)(DataKey("cip-phone-number-verification"), phoneNumberAndPasscode)
+      .map(_ => phoneNumberAndPasscode)
   }
 
   def retrievePasscode(phoneNumber: String): Future[Option[PhoneNumberPasscodeData]] = {
-    logger.debug(s"Retrieving phoneNumberAndOtp from database for $phoneNumber")
+    logger.debug(s"Retrieving phoneNumberAndPasscode from database for $phoneNumber")
```
Comment (johno1985) [inline app/uk/gov/hmrc/cipphonenumberverification/services/PasscodeService.scala:36]: and this, PII

File: app/uk/gov/hmrc/cipphonenumberverification/services/VerifyHelper.scala [hunk 3]
```diff
@@ -88,18 +88,20 @@ abstract class VerifyHelper @Inject()(otpService: OtpService,
...
-    val dataToSave = new PhoneNumberPasscodeData(validatedPhoneNumber.phoneNumber, otp, now)
+    val dataToSave = new PhoneNumberPasscodeData(validatedPhoneNumber.phoneNumber, passcode, now)
     auditService.sendExplicitAuditEvent(PhoneNumberVerificationRequest,
-      VerificationRequestAuditEvent(dataToSave.phoneNumber, otp))
+      VerificationRequestAuditEvent(dataToSave.phoneNumber, passcode))
 
     passcodeService.persistPasscode(dataToSave) transformWith {
-      case Success(savedPhoneNumberPasscodeData) => sendPasscode(savedPhoneNumberPasscodeData)
+      case Success(savedPhoneNumberPasscodeData) =>
+        println(savedPhoneNumberPasscodeData)
```
Comment (johno1985) [inline app/uk/gov/hmrc/cipphonenumberverification/services/VerifyHelper.scala:99]: left by mistake?

**Review by johno1985 — CHANGES_REQUESTED** [review]
minor comments

**Review by johno1985 — APPROVED** [review]

**Pull request merged**